rgf/test/test.py

```diff
@@ -73,6 +71,17 @@ def test_regressor(self):
         print("mse: " + str(mse))
         assert_less(mse, 6.0)
 
+    def test_sample_weight(self):
+        X_train, X_test, y_train, y_test = train_test_split(self.iris.data,
+                                                            self.iris.target,
+                                                            test_size=0.2,
+                                                            random_state=42)
+
+        clf = RGFClassifier()
+        y_pred = clf.fit(X_train, y_train).predict_proba(X_test)
+        y_pred_weighted = clf.fit(X_train, y_train, np.ones(y_train.shape[0])).predict_proba(X_test)
+        self.assertTrue(np.isclose(y_pred, y_pred_weighted).all())
```

Comment:
I think that 2 tests in one function will be OK.

About zero inputs: in my opinion we should think about it a bit more later and now leave it as is. Because, as for me, I'm not sure that I'll be able to fix C++ code (or maybe this is feature, not bug! Who knows?.. 😄 )